File: pinyougou-web-manager/src/main/webapp/js/service/seckillService.js
//服务层
app.service('seckillService',function($http){
	    	
	//读取列表数据绑定到表单中
	//删除
	this.dele=function(ids){
		return $http.get('../seckill/delete.do?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../seckill/search.do?page='+page+"&rows="+rows, searchEntity);
	}    
	//审核
	this.updateStatus = function(ids,status){
		return $http.get('../seckill/updateStatus.do?ids='+ids+"&status="+status);
	}
});

File: pinyougou-interface/src/main/java/cn/itcast/core/service/SeckillService.java
package cn.itcast.core.service;

import cn.itcast.core.pojo.seckill.SeckillGoods;
import cn.itcast.core.pojo.seckill.SeckillOrder;
import entity.PageResult;

public interface SeckillService {

    PageResult search(Integer page, Integer rows, SeckillOrder seckillOrder);

    void delete(Long[] ids);

    void updateStatus(Long[] ids, String status);

    PageResult searchGoods(Integer page, Integer rows, SeckillGoods seckillGoods);
}
